src/lib/viewers/BaseViewer.js

```diff
@@ -1161,6 +1162,7 @@ class BaseViewer extends EventEmitter {
             addedElement = this.containerEl.insertBefore(element, firstChildEl);
         }
 
+        addedElement.className = classes;
```

Comment:
This feels a little shoehorned. Having each viewer just assign `className` in the same way as they do `tabIndex` seemed fine to me.